mmv1/third_party/terraform/services/composer/resource_composer_environment_test.go.erb

```diff
@@ -1081,6 +1082,93 @@ func testAccComposerEnvironmentDestroyProducer(t *testing.T) func(s *terraform.S
 	}
 }
 
+// Checks environment creation with custom bucket
+func TestAccComposerEnvironment_customBucket(t *testing.T) {
+	t.Parallel()
+
+	bucketName := fmt.Sprintf("%s-%d", testComposerBucketPrefix, acctest.RandInt(t))
+	envName := fmt.Sprintf("%s-%d", testComposerEnvironmentPrefix, acctest.RandInt(t))
+	network := fmt.Sprintf("%s-%d", testComposerNetworkPrefix, acctest.RandInt(t))
+	subnetwork := network + "-1"
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:		          func() { acctest.AccTestPreCheck(t) },
```

Comment:
nit: Line indentation should be tabs, but internal alignment should be spaces. Mostly flagging because a few lines here use both tabs+spaces for alignment.